requirements/edx/base.txt

```diff
@@ -129,7 +129,6 @@ help-tokens==1.1.2        # via -r requirements/edx/base.in
 html5lib==1.1             # via -r requirements/edx/base.in, ora2
 icalendar==4.0.7          # via -r requirements/edx/base.in
 idna==2.10                # via -r requirements/edx/paver.txt, requests
-importlib-metadata==1.7.0  # via -c requirements/edx/../constraints.txt, -r requirements/edx/paver.txt, kombu, path
```

Comment:
I plan on ignoring this and approving, but I can't figure out why the `make upgrade` job in Jenkins is doing something different than devstack, and why the flip-flop on these. I have the same issue.
FYI: @jmbowman @timmc-edx @edx/arbi-bom

Reply:
the `make upgrade` Jenkins job is still using python 3.5 because we haven't yet dropped python 3.5 in edx-platform while the devstack has been updated to use python 3.8 that's why there's a difference

Reply:
Thanks @mraarif. A related issue is that when I try running `make upgrade` from devstack under a python 3.5 virtualenv, I run into a error. Maybe I don’t have the right version of pip or pip-tools or something? It would be nice to have simple instructions for what to do from now until we have upgraded the job to python 3.8.

Reply:
FYI: This has been resolved. We are now using Python 3.8 everywhere.